ClinGen/clincoded
# Connects to #800. External data source crediting.

This PR consists of implementation pertinent to adding UIs for crediting external data sources such as myvariant.info and VEP.

**Steps to test:**
1. Login and select 55962 variant id.
2. On the Basic Info tab, expect to see the "VEP" icons next to the headers of **RefSeq Transcripts** and **Ensembl Transcripts** tables. Expect to see the "VEP" credit at the bottom of the page.
3. On the Population tab, expect to see the "MVi" icons next to the headers of **ExAC** and **ESP** tables, and the "VEP" icon next to the header of **1000 Genomes** table. Expect to see the "MVi" and "VEP" credits at the bottom of the page.
3. On the Predictors tab, expect to see the "MVi" icons next to the headers of **Other Predictors** and **Conservation Analysis** tables. Expect to see the "MVi" credit at the bottom of the page.
